Auto update failing for p11-kit

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [p11-kit](https://github.com/termux/termux-packages/blob/501b09d27c7fc95b59e1a7c48193bb5312d04639/packages/p11-kit) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating p11-kit [Current version: 0.25.9]
INFO: package being updated to 0.25.10.
Downloading https://github.com/p11-glue/p11-kit/releases/download/0.25.10/p11-kit-0.25.10.tar.xz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 1028k  100 1028k    0     0  2714k      0 --:--:-- --:--:-- --:--:-- 2714k

You are about to commit these changes:

--------------------
diff --git a/packages/p11-kit/build.sh b/packages/p11-kit/build.sh
index 7e557cd15b..ec3737e7de 100644
--- a/packages/p11-kit/build.sh
+++ b/packages/p11-kit/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE="https://p11-glue.github.io/p11-glue/p11-kit.html"
 TERMUX_PKG_DESCRIPTION="Provides a way to load and enumerate PKCS#11 modules"
 TERMUX_PKG_LICENSE="BSD 3-Clause"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.25.9"
+TERMUX_PKG_VERSION="0.25.10"
 TERMUX_PKG_SRCURL="https://github.com/p11-glue/p11-kit/releases/download/$TERMUX_PKG_VERSION/p11-kit-$TERMUX_PKG_VERSION.tar.xz"
-TERMUX_PKG_SHA256=f6512a10b2dcf2651cfd57dd767a36c6e44494ab37724c10d4304fe9f0a36497
+TERMUX_PKG_SHA256=a62a137a966fb3a9bbfa670b4422161e369ddea216be51425e3be0ab2096e408
 TERMUX_PKG_DEPENDS="libffi, libtasn1"
 TERMUX_PKG_BUILD_DEPENDS="aosp-libs, bash-completion"
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
--------------------

bump(main/p11-kit): 0.25.10


Not committing to Git!

INFO: Trying to build package.
INFO: Package p11-kit exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building p11-kit for arch aarch64...
::group::INFO: Fetching repo metadata
[packages-cf.termux.dev-apt-termux-root]: Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
[packages-cf.termux.dev-apt-termux-root]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-root]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-x11]: Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
[packages-cf.termux.dev-apt-termux-x11]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-x11]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-main]: Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
[packages-cf.termux.dev-apt-termux-main]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-main]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-main]: 
[packages-cf.termux.dev-apt-termux-main]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-main]: 100 13134  100 13134    0     0   173k      0 --:--:-- --:--:-- --:--:--  175k
[packages-cf.termux.dev-apt-termux-main]: Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
[packages-cf.termux.dev-apt-termux-main]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-main]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-root]: 
[packages-cf.termux.dev-apt-termux-root]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-root]:   0 13320    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-root]: 100 13320  100 13320    0     0  22713      0 --:--:-- --:--:-- --:--:-- 22691
[packages-cf.termux.dev-apt-termux-root]: Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
[packages-cf.termux.dev-apt-termux-root]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-root]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-main]: 
[packages-cf.termux.dev-apt-termux-main]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-main]: 100   833  100   833    0     0   1469      0 --:--:-- --:--:-- --:--:--  1469
[packages-cf.termux.dev-apt-termux-main]: 100   833  100   833    0     0   1469      0 --:--:-- --:--:-- --:--:--  1469
[packages-cf.termux.dev-apt-termux-main]: gpg: Signature made Mon 22 Sep 2025 09:13:21 AM UTC
[packages-cf.termux.dev-apt-termux-main]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-main]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-main]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
[packages-cf.termux.dev-apt-termux-x11]: 
[packages-cf.termux.dev-apt-termux-x11]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-x11]: 100 13120  100 13120    0     0  25354      0 --:--:-- --:--:-- --:--:-- 25328
[packages-cf.termux.dev-apt-termux-x11]: 100 13120  100 13120    0     0  25350      0 --:--:-- --:--:-- --:--:-- 25328
[packages-cf.termux.dev-apt-termux-x11]: Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
[packages-cf.termux.dev-apt-termux-x11]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-x11]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-root]: 
[packages-cf.termux.dev-apt-termux-root]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-root]: 100   833  100   833    0     0   1431      0 --:--:-- --:--:-- --:--:--  1433
[packages-cf.termux.dev-apt-termux-root]: gpg: Signature made Sun 21 Sep 2025 12:32:34 PM UTC
[packages-cf.termux.dev-apt-termux-root]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-root]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-root]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
[packages-cf.termux.dev-apt-termux-x11]: 
[packages-cf.termux.dev-apt-termux-x11]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-x11]: 100   833  100   833    0     0   1578      0 --:--:-- --:--:-- --:--:--  1580
[packages-cf.termux.dev-apt-termux-x11]: gpg: Signature made Mon 22 Sep 2025 08:00:20 AM UTC
[packages-cf.termux.dev-apt-termux-x11]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-x11]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-x11]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
::endgroup::
Downloading dependency aosp-libs@9.0.0-r76-1 if necessary...
Found aosp-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/a/aosp-libs/aosp-libs_9.0.0-r76-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0 9424k    0 32427    0     0  43397      0  0:03:42 --:--:--  0:03:42 43351
  4 9424k    4  415k    0     0   290k      0  0:00:32  0:00:01  0:00:31  290k
100 9424k  100 9424k    0     0  4401k      0  0:00:02  0:00:02 --:--:-- 4402k
extracting aosp-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency aosp-utils@9.0.0-r76-1 if necessary...
Found aosp-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/a/aosp-utils/aosp-utils_9.0.0-r76-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  9  319k    9 32429    0     0  45702      0  0:00:07 --:--:--  0:00:07 45674
100  319k  100  319k    0     0   267k      0  0:00:01  0:00:01 --:--:--  267k
extracting aosp-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-support@29-1 if necessary...
Found libandroid-support in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/liba/libandroid-support/libandroid-support_29-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 10880  100 10880    0     0   205k      0 --:--:-- --:--:-- --:--:--  208k
extracting libandroid-support to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libiconv@1.18-1 if necessary...
Found libiconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libi/libiconv/libiconv_1.18-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  548k  100  548k    0     0  6550k      0 --:--:-- --:--:-- --:--:-- 6602k
extracting libiconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency iconv@1.18-1 if necessary...
Found iconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/i/iconv/iconv_1.18-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 10024  100 10024    0     0   180k      0 --:--:-- --:--:-- --:--:--  181k
extracting iconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses@6.5.20240831-3 if necessary...
Found ncurses in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses/ncurses_6.5.20240831-3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  524k  100  524k    0     0  6493k      0 --:--:-- --:--:-- --:--:-- 6561k
extracting ncurses to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-utils@6.5.20240831-3 if necessary...
Found ncurses-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-utils/ncurses-utils_6.5.20240831-3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 87932  100 87932    0     0  1402k      0 --:--:-- --:--:-- --:--:-- 1407k
extracting ncurses-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs@6.5.20240831-3 if necessary...
Found ncurses-ui-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-ui-libs/ncurses-ui-libs_6.5.20240831-3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 32944  100 32944    0     0   556k      0 --:--:-- --:--:-- --:--:--  564k
extracting ncurses-ui-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs-static@6.5.20240831-3 if necessary...
Found ncurses-ui-libs-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-ui-libs-static/ncurses-ui-libs-static_6.5.20240831-3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 39584  100 39584    0     0   688k      0 --:--:-- --:--:-- --:--:--  690k
extracting ncurses-ui-libs-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency readline@8.3.1-1 if necessary...
Found readline in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/r/readline/readline_8.3.1-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  286k  100  286k    0     0  4158k      0 --:--:-- --:--:-- --:--:-- 4216k
extracting readline to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libbz2@1.0.8-8 if necessary...
Found libbz2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libb/libbz2/libbz2_1.0.8-8_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 26116  100 26116    0     0   492k      0 --:--:-- --:--:-- --:--:--  500k
extracting libbz2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency bzip2@1.0.8-8 if necessary...
Found bzip2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/b/bzip2/bzip2_1.0.8-8_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 26152  100 26152    0     0   467k      0 --:--:-- --:--:-- --:--:--  472k
extracting bzip2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pcre2@10.46 if necessary...
Found pcre2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/pcre2/pcre2_10.46_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  925k  100  925k    0     0  10.8M      0 --:--:-- --:--:-- --:--:-- 11.0M
extracting pcre2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pcre2grep@10.46 if necessary...
Found pcre2grep in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/pcre2grep/pcre2grep_10.46_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 34004  100 34004    0     0   610k      0 --:--:-- --:--:-- --:--:--  614k
extracting pcre2grep to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-selinux@14.0.0.11-1 if necessary...
Found libandroid-selinux in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/liba/libandroid-selinux/libandroid-selinux_14.0.0.11-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 59624  100 59624    0     0  1039k      0 --:--:-- --:--:-- --:--:-- 1058k
extracting libandroid-selinux to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libc++@28c if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libgmp@6.3.0-2 if necessary...
Found libgmp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libg/libgmp/libgmp_6.3.0-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  320k  100  320k    0     0  4327k      0 --:--:-- --:--:-- --:--:-- 4333k
extracting libgmp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ca-certificates@1:2025.09.09 if necessary...
Found ca-certificates in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates to /home/builder/.termux-build/_cache-all...
Downloading dependency ca-certificates-java@1:2025.09.09 if necessary...
Found ca-certificates-java in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates-java to /home/builder/.termux-build/_cache-all...
Downloading dependency zlib@1.3.1-1 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl@1:3.5.2 if necessary...
Found openssl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl-tool@1:3.5.2 if necessary...
Found openssl-tool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl-tool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency coreutils@9.7-3 if necessary...
Found coreutils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/c/coreutils/coreutils_9.7-3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  745k  100  745k    0     0  9180k      0 --:--:-- --:--:-- --:--:-- 9093k
extracting coreutils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libnghttp2@1.67.1 if necessary...
Found libnghttp2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libnghttp2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libnghttp3@1.11.0-1 if necessary...
Found libnghttp3 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libnghttp3 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libssh2@1.11.1-1 if necessary...
Found libssh2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libssh2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcurl@8.16.0 if necessary...
Found libcurl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libcurl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency curl@8.16.0 if necessary...
Found curl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting curl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency dash@0.5.12-1 if necessary...
Found dash in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/d/dash/dash_0.5.12-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 65828  100 65828    0     0  1100k      0 --:--:-- --:--:-- --:--:-- 1108k
extracting dash to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency dialog@1.3-20240307-1 if necessary...
Found dialog in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/d/dialog/dialog_1.3-20240307-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   97k  100   97k    0     0  1597k      0 --:--:-- --:--:-- --:--:-- 1603k
extracting dialog to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency diffutils@3.12-2 if necessary...
Found diffutils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/d/diffutils/diffutils_3.12-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  158k  100  158k    0     0  2458k      0 --:--:-- --:--:-- --:--:-- 2482k
extracting diffutils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency findutils@4.10.0-1 if necessary...
Found findutils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/f/findutils/findutils_4.10.0-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  245k  100  245k    0     0  3542k      0 --:--:-- --:--:-- --:--:-- 3557k
extracting findutils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libmpfr@4.2.1-1 if necessary...
Found libmpfr in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libm/libmpfr/libmpfr_4.2.1-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  265k  100  265k    0     0  3818k      0 --:--:-- --:--:-- --:--:-- 3851k
extracting libmpfr to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gawk@5.3.1-2 if necessary...
Found gawk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/g/gawk/gawk_5.3.1-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  763k  100  763k    0     0  9087k      0 --:--:-- --:--:-- --:--:-- 9200k
extracting gawk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency grep@3.12-2 if necessary...
Found grep in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/g/grep/grep_3.12-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  125k  100  125k    0     0  1977k      0 --:--:-- --:--:-- --:--:-- 1984k
extracting grep to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gzip@1.14-1 if necessary...
Found gzip in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/g/gzip/gzip_1.14-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 87548  100 87548    0     0  1382k      0 --:--:-- --:--:-- --:--:-- 1401k
extracting gzip to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency less@679-2 if necessary...
Found less in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/l/less/less_679-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  126k  100  126k    0     0  2064k      0 --:--:-- --:--:-- --:--:-- 2080k
extracting less to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency procps@3.3.17-6 if necessary...
Found procps in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/procps/procps_3.3.17-6_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  139k  100  139k    0     0  2217k      0 --:--:-- --:--:-- --:--:-- 2250k
extracting procps to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency psmisc@23.7-1 if necessary...
Found psmisc in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/psmisc/psmisc_23.7-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 41188  100 41188    0     0   638k      0 --:--:-- --:--:-- --:--:--  648k
extracting psmisc to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency sed@4.9-2 if necessary...
Found sed in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/s/sed/sed_4.9-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  114k  100  114k    0     0  1811k      0 --:--:-- --:--:-- --:--:-- 1823k
extracting sed to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-glob@0.6-3 if necessary...
Found libandroid-glob in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/liba/libandroid-glob/libandroid-glob_0.6-3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  7032  100  7032    0     0   130k      0 --:--:-- --:--:-- --:--:--  132k
extracting libandroid-glob to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tar@1.35-1 if necessary...
Found tar in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/t/tar/tar_1.35-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  334k  100  334k    0     0  4499k      0 --:--:-- --:--:-- --:--:-- 4457k
extracting tar to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency termux-am@0.8.0-2 if necessary...
Found termux-am in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/t/termux-am/termux-am_0.8.0-2_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  563k  100  563k    0     0  6559k      0 --:--:-- --:--:-- --:--:-- 6632k
extracting termux-am to /home/builder/.termux-build/_cache-all...
Downloading dependency termux-am-socket@1.5.0-1 if necessary...
Found termux-am-socket in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/t/termux-am-socket/termux-am-socket_1.5.0-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 16132  100 16132    0     0   285k      0 --:--:-- --:--:-- --:--:--  286k
extracting termux-am-socket to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency termux-core@0.4.0-1 if necessary...
Found termux-core in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/t/termux-core/termux-core_0.4.0-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  193k  100  193k    0     0  2564k      0 --:--:-- --:--:-- --:--:-- 2581k
extracting termux-core to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency termux-core-static@0.4.0-1 if necessary...
Found termux-core-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/t/termux-core-static/termux-core-static_0.4.0-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 20176  100 20176    0     0   335k      0 --:--:-- --:--:-- --:--:--  339k
extracting termux-core-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency termux-exec@1:2.4.0-1 if necessary...
Found termux-exec in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/t/termux-exec/termux-exec_1:2.4.0-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  281k  100  281k    0     0  3972k      0 --:--:-- --:--:-- --:--:-- 4019k
extracting termux-exec to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libblkid@2.40.2-5 if necessary...
Found libblkid in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libb/libblkid/libblkid_2.40.2-5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  121k  100  121k    0     0  1959k      0 --:--:-- --:--:-- --:--:-- 1985k
extracting libblkid to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libmount@2.40.2-5 if necessary...
Found libmount in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libm/libmount/libmount_2.40.2-5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  142k  100  142k    0     0  2093k      0 --:--:-- --:--:-- --:--:-- 2097k
extracting libmount to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsmartcols@2.40.2-5 if necessary...
Found libsmartcols in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libs/libsmartcols/libsmartcols_2.40.2-5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 98000  100 98000    0     0  1468k      0 --:--:-- --:--:-- --:--:-- 1450k
extracting libsmartcols to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libuuid@2.40.2-5 if necessary...
Found libuuid in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libu/libuuid/libuuid_2.40.2-5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 14968  100 14968    0     0   281k      0 --:--:-- --:--:-- --:--:--  286k
extracting libuuid to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libfdisk@2.40.2-5 if necessary...
Found libfdisk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libf/libfdisk/libfdisk_2.40.2-5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 21  152k   21 33789    0     0  38426      0  0:00:04 --:--:--  0:00:04 38396
100  152k  100  152k    0     0   146k      0  0:00:01  0:00:01 --:--:--  146k
extracting libfdisk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcap-ng@2:0.8.5-1 if necessary...
Found libcap-ng in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libc/libcap-ng/libcap-ng_2:0.8.5-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 35672  100 35672    0     0   649k      0 --:--:-- --:--:-- --:--:--  657k
extracting libcap-ng to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency util-linux@2.40.2-5 if necessary...
Found util-linux in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/u/util-linux/util-linux_2.40.2-5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  688k  100  688k    0     0  8566k      0 --:--:-- --:--:-- --:--:-- 8601k
extracting util-linux to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency uuid-utils@2.40.2-5 if necessary...
Found uuid-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/u/uuid-utils/uuid-utils_2.40.2-5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 40744  100 40744    0     0   754k      0 --:--:-- --:--:-- --:--:--  765k
extracting uuid-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency mount-utils@2.40.2-5 if necessary...
Found mount-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/m/mount-utils/mount-utils_2.40.2-5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 24  129k   24 32429    0     0  47709      0  0:00:02 --:--:--  0:00:02 47689
100  129k  100  129k    0     0   131k      0 --:--:-- --:--:-- --:--:--  131k
extracting mount-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency fdisk@2.40.2-5 if necessary...
Found fdisk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/f/fdisk/fdisk_2.40.2-5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  109k  100  109k    0     0  1805k      0 --:--:-- --:--:-- --:--:-- 1827k
extracting fdisk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency blk-utils@2.40.2-5 if necessary...
Found blk-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/b/blk-utils/blk-utils_2.40.2-5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  172k  100  172k    0     0  2421k      0 --:--:-- --:--:-- --:--:-- 2458k
extracting blk-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency liblzma@5.8.1-1 if necessary...
Found liblzma in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libl/liblzma/liblzma_5.8.1-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  187k  100  187k    0     0  2751k      0 --:--:-- --:--:-- --:--:-- 2750k
extracting liblzma to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xz-utils@5.8.1-1 if necessary...
Found xz-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/x/xz-utils/xz-utils_5.8.1-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 71104  100 71104    0     0  1282k      0 --:--:-- --:--:-- --:--:-- 1285k
extracting xz-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency termux-tools@1.46.0+really1.45.0-1 if necessary...
Found termux-tools in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/t/termux-tools/termux-tools_1.46.0+really1.45.0-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 33592  100 33592    0     0   624k      0 --:--:-- --:--:-- --:--:--  630k
extracting termux-tools to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency bash@5.3.3-1 if necessary...
Found bash in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/b/bash/bash_5.3.3-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  933k  100  933k    0     0  10.6M      0 --:--:-- --:--:-- --:--:-- 10.7M
extracting bash to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency bash-completion@2.16.0 if necessary...
Found bash-completion in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/b/bash-completion/bash-completion_2.16.0_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  179k  100  179k    0     0  2633k      0 --:--:-- --:--:-- --:--:-- 2645k
extracting bash-completion to /home/builder/.termux-build/_cache-all...
Downloading dependency libffi@3.4.7-1 if necessary...
Found libffi in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libf/libffi/libffi_3.4.7-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 30672  100 30672    0     0   594k      0 --:--:-- --:--:-- --:--:--  599k
extracting libffi to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libtasn1@4.20.0-1 if necessary...
Found libtasn1 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libt/libtasn1/libtasn1_4.20.0-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   99k  100   99k    0     0  1672k      0 --:--:-- --:--:-- --:--:-- 1682k
extracting libtasn1 to /home/builder/.termux-build/_cache-aarch64...
Downloading p11-kit@0.25.10 source from 'https://github.com/p11-glue/p11-kit/releases/download/0.25.10/p11-kit-0.25.10.tar.xz' if necessary...
Downloading https://github.com/p11-glue/p11-kit/releases/download/0.25.10/p11-kit-0.25.10.tar.xz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 1028k  100 1028k    0     0  3235k      0 --:--:-- --:--:-- --:--:-- 3235k
Downloading https://github.com/proot-me/proot/releases/download/v5.3.0/proot-v5.3.0-x86_64-static
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 1856k  100 1856k    0     0  4789k      0 --:--:-- --:--:-- --:--:-- 4789k
Downloading https://github.com/multiarch/qemu-user-static/releases/download/v7.2.0-1/qemu-aarch64-static
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  9.9M  100  9.9M    0     0  14.3M      0 --:--:-- --:--:-- --:--:-- 14.3M
patching file common/meson.build
patching file meson.build
configure: error: unrecognized option: '-Dtrust_module=enabled'
Try '/home/builder/.termux-build/p11-kit/src/configure --help' for more information
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2025-09-22 12:18:54 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/17914566784">17914566784</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/p11-kit/build.sh
+++ b/packages/p11-kit/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE="https://p11-glue.github.io/p11-glue/p11-kit.html"
 TERMUX_PKG_DESCRIPTION="Provides a way to load and enumerate PKCS#11 modules"
 TERMUX_PKG_LICENSE="BSD 3-Clause"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.25.9"
+TERMUX_PKG_VERSION="0.25.10"
 TERMUX_PKG_SRCURL="https://github.com/p11-glue/p11-kit/releases/download/$TERMUX_PKG_VERSION/p11-kit-$TERMUX_PKG_VERSION.tar.xz"
-TERMUX_PKG_SHA256=f6512a10b2dcf2651cfd57dd767a36c6e44494ab37724c10d4304fe9f0a36497
+TERMUX_PKG_SHA256=a62a137a966fb3a9bbfa670b4422161e369ddea216be51425e3be0ab2096e408
 TERMUX_PKG_DEPENDS="libffi, libtasn1"
 TERMUX_PKG_BUILD_DEPENDS="aosp-libs, bash-completion"
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
@@ -13,6 +13,9 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 TERMUX_PKG_AUTO_UPDATE=true
 
 termux_step_pre_configure() {
+	# force meson
+	rm configure
+
 	if [[ "$TERMUX_ON_DEVICE_BUILD" == "false" ]]; then
 		termux_setup_proot
 		sed \

